diff --git a/src/libs/browser/CMakeLists.txt b/src/libs/browser/CMakeLists.txt
index a267d1a..b99e85e 100644
--- a/src/libs/browser/CMakeLists.txt
+++ b/src/libs/browser/CMakeLists.txt
@@ -7,5 +7,8 @@ add_library(Browser STATIC
 
 target_link_libraries(Browser)
 
+list(APPEND CMAKE_PREFIX_PATH /usr/local/opt/qt)
+set(Qt5WebKit_DIR /usr/local/opt/qt5-webkit/lib/cmake/Qt5WebKit)
+set(Qt5WebKitWidgets_DIR /usr/local/opt/qt5-webkit/lib/cmake/Qt5WebKitWidgets)
 find_package(Qt5 COMPONENTS WebKitWidgets REQUIRED)
 target_link_libraries(Browser Qt5::WebKitWidgets)
diff --git a/src/libs/core/CMakeLists.txt b/src/libs/core/CMakeLists.txt
index 3bd7371..9a75214 100644
--- a/src/libs/core/CMakeLists.txt
+++ b/src/libs/core/CMakeLists.txt
@@ -9,9 +9,14 @@ add_library(Core STATIC
 
 target_link_libraries(Core Registry Ui)
 
+list(APPEND CMAKE_PREFIX_PATH /usr/local/opt/qt)
+set(Qt5WebKit_DIR /usr/local/opt/qt5-webkit/lib/cmake/Qt5WebKit)
+set(Qt5WebKitWidgets_DIR /usr/local/opt/qt5-webkit/lib/cmake/Qt5WebKitWidgets)
 find_package(Qt5 COMPONENTS Network WebKit Widgets REQUIRED)
 target_link_libraries(Core Qt5::Network Qt5::WebKit Qt5::Widgets)
 
+list(APPEND CMAKE_PREFIX_PATH /usr/local/opt/libarchive)
+list(APPEND CMAKE_LIBRARY_PATH /usr/local/opt/libarchive)
 find_package(LibArchive REQUIRED)
 include_directories(${LibArchive_INCLUDE_DIRS})
 target_link_libraries(Core ${LibArchive_LIBRARIES})
diff --git a/src/libs/ui/CMakeLists.txt b/src/libs/ui/CMakeLists.txt
index 8f4b078..68cc2b2 100644
--- a/src/libs/ui/CMakeLists.txt
+++ b/src/libs/ui/CMakeLists.txt
@@ -24,5 +24,8 @@ add_library(Ui STATIC
 
 target_link_libraries(Ui Browser Sidebar QxtGlobalShortcut Widgets Registry)
 
+list(APPEND CMAKE_PREFIX_PATH /usr/local/opt/qt)
+set(Qt5WebKit_DIR /usr/local/opt/qt5-webkit/lib/cmake/Qt5WebKit)
+set(Qt5WebKitWidgets_DIR /usr/local/opt/qt5-webkit/lib/cmake/Qt5WebKitWidgets)
 find_package(Qt5 COMPONENTS WebKitWidgets REQUIRED)
 target_link_libraries(Ui Qt5::WebKitWidgets)
